commit 69d568b7ff8f6adae556a84d98c3f58bfe41da4f
Author: Nathan Chancellor <natechancellor@gmail.com>
Date:   Sun Mar 4 20:09:26 2018 -0700

    usb: dwb_otg: Fix unreachable switch statement warning
    
    This warning appears with GCC 7.3.0 from toolchains.bootlin.com:
    
    ../drivers/usb/host/dwc_otg/dwc_otg_fiq_fsm.c: In function ‘fiq_fsm_update_hs_isoc’:
    ../drivers/usb/host/dwc_otg/dwc_otg_fiq_fsm.c:595:61: warning: statement will never be executed [-Wswitch-unreachable]
       st->hctsiz_copy.b.xfersize = nrpackets * st->hcchar_copy.b.mps;
                                                ~~~~~~~~~~~~~~~~~^~~~
    
    Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>

diff --git a/drivers/usb/host/dwc_otg/dwc_otg_fiq_fsm.c b/drivers/usb/host/dwc_otg/dwc_otg_fiq_fsm.c
index 2f96043d05b3..0e043a20d1df 100644
--- a/drivers/usb/host/dwc_otg/dwc_otg_fiq_fsm.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_fiq_fsm.c
@@ -591,8 +591,8 @@ static int notrace noinline fiq_fsm_update_hs_isoc(struct fiq_state *state, int
 		}
 
 	} else {
-		switch (st->hcchar_copy.b.multicnt) {
 		st->hctsiz_copy.b.xfersize = nrpackets * st->hcchar_copy.b.mps;
+		switch (st->hcchar_copy.b.multicnt) {
 		case 1:
 			st->hctsiz_copy.b.pid = DWC_PID_DATA0;
 			break;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1157_69d568b7ff8f6adae556a84d98c3f58bfe41da4f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1157_69d568b7ff8f6adae556a84d98c3f58bfe41da4f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
